orchestra/models.py

```diff
@@ -265,7 +268,7 @@ class Status:
         (Status.PROCESSING, 'Processing'),
         (Status.SUBMITTED, 'Submitted'))
 
-    start_datetime = models.DateTimeField(auto_now_add=True)
+    start_datetime = models.DateTimeField(default=timezone.now)
```

Comment:
why would the old ones be dirty? Does auto_now_add not add timezone-aware datetimes?